gk/main.c

```diff
@@ -251,21 +299,22 @@ cycle_from_second(uint64_t time)
 }
 
 static int
-gk_process_granted(struct flow_entry *fe, struct ipacket *packet)
+gk_process_granted(struct flow_entry *fe,
+	struct ipacket *packet, struct gk_lpm *ltbl)
 {
 	int ret;
 	bool renew_cap;
 	uint8_t priority = PRIORITY_GRANTED;
 	uint64_t now = rte_rdtsc();
 	struct rte_mbuf *pkt = packet->pkt;
 
-	/* TODO The tunnel information should come from the LPM table. */
-	struct ipip_tunnel_info tunnel;
-	memset(&tunnel, 0, sizeof(tunnel));
+	/* The tunnel information should come from the LPM table. */
```

Comment:
Same thing for this comment -- it's not adding any useful information beyond what the code says.